src/components/Sum.jsx

```diff
@@ -1 +1,19 @@
 // Don't forget to add default values an check types of props
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Sum = ({ a = 0, b = 0 }) => (
```

Comment:
```suggestion
export const Sum = ({ a = 0, b = 0 }) => (
```
You could use defaultProps for this instead